target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -225,6 +229,10 @@ case "$FIRMWARE" in
 	qxwlan,e2600ac-c2)
 		ath10kcal_extract "0:ART" 0x5000 0x2f20
 		;;
+	engenius,emr3500)
+		ath10kcal_extract "0:ART" 0x5000 0x2f20
+		ath10kcal_patch_mac_crc $(macaddr_add "$(cat /sys/class/net/eth0/address)" 1)
```

Comment:
It's not stored in 0:APPSBLENV.

Reply:
The stock firmware uses (LAN MAC address + 1) as 5G wifi MAC address.

Reply:
Okay. Is eth0 the same as one of $(mtd_get_mac_ascii 0:APPSBLENV wanaddr) or $(mtd_get_mac_ascii 0:APPSBLENV ethaddr)?
In that case, I would prefer patching WiFi MAC based on those instead of $(cat /sys/class/net/eth0/address) ...

Reply:
OK. done.
I also use this way to patch MAC address for 2.4G above.